.gitignore

```diff
@@ -36,9 +36,7 @@ pom.xml.versionsBackup
 # hprof dumps
 /*.hprof
 # ignore 'randomly' strewn around logs
-server.log
 # ignore java crashes
-hs_err_pid*.log
```

Comment:
There is already a *.log on line 57.

Reply:
Should probably also remove the comments `# ignore 'randomly' strewn around logs` and `# ignore java crashes`.